docs/src/examples/counter.md

```diff
@@ -0,0 +1,117 @@
+# Counter
+
+> In this example project we'll walk through how to run a Fuel Indexer service that uses WASM-based execution. We'll go through project structure, how to actually use the service, as well as a few key concepts.
```

Comment:
This line states that the project uses WASM-based execution, but the last line states that the project used native Rust execution.

Reply:
Ahh good catch @deekerno. Pushing a change